azplugins/test-py/test_bond_fene.py

```diff
@@ -117,6 +117,33 @@ def test_potential(self):
         self.assertAlmostEqual(f1[1],0)
         self.assertAlmostEqual(f1[2],0)
 
+    # test the calculation of force and potential when delta is non-zero
+    def test_potential_delta_nonzero(self):
+        #fene = azplugins.bond.fene()
+        fene = md.bond.fene()
```

Comment:
This is testing the HOOMD fene bond, not the azplugins one. Please remove the `md.bond` line and uncomment the `azplugins.bond` line.

Reply:
Hi Mike,
In the already existing test for potential and force in test_bond_fene.py (I am talking about this function --> test_potential), why did you comment out fene=azplugins.bond.fene(). Aren't we testing for the case when delta=0 for the azplugin's fene bond here? Should I keep it like that or should I change it?

Reply:
You found a mistake in the tests! Could you please apply the same correction in `test_potential`? We want to test the `azplugins.bond.fene` class and not `hoomd.md.bond.fene`. Probably the original developer used both versions to implement the test, and just forgot to switch to the right one. (And we missed it in code review).